index.html

```diff
@@ -28,7 +28,6 @@ <h1>Handcrafted, home-made masterpieces</h1>
       height="315"
       src="https://www.youtube.com/embed/y8Yv4pnO7qc"
       title="YouTube video player"
-      frameborder="0"
```

Comment:
- frameBorder="0" is obsolete and it's best practice to use CSS instead. 😃

Reply:
This is a nice comment however, It would be better to provide additional information about the CSS you would want him to use. Providing a link to the CSS you suggested would help the author to make research and be able to implement the required changes.